www/js/controller.js

```diff
@@ -188,10 +194,16 @@ const macMenu = [
         role: 'about',
       },
       ...updateMenu,
+      {
+        label: 'Guide...',
```

Comment:
Same as above - under a new Help top-level menu

Reply:
Could this please be moved to match macOS menubar? With a top-level menu called Help